packages/react-native-tracer-provider/src/types.ts

```diff
@@ -0,0 +1,22 @@
+import {TracerProvider} from "@opentelemetry/api";
+
+export interface UseEmbraceNativeTracerProviderResult {
```

Comment:
can we remove the `Use` prefix from the name for this interface?

Reply:
I wasn't sure about the convention here, this is the type of the return value of `useEmbraceNativeTracerProvider` so I was naming it to match, what do you think?

Reply:
I think the convention is remove the use and just let EmbraceNativeTracerProvider (A least I saw this way in other sdks)

Reply:
I usually name this kind of instances as `EmbraceNativeTracerProviderReturn` (or `EmbraceNativeTracerProviderResult` as you have here but without the prefix)

Reply:
updated